mem-reserve.1

```diff
@@ -27,11 +27,19 @@ Computes the kernel parameters necessary to reserve BYTES memory at
 the top of the physical memory map. BYTES may be suffixed with P, T,
 G, M or K to indicate a scaling factor.
 .RE
+.PP
+.B -b BYTES, --bsize BYTES
+.RS 4
+Show LFS cfg declarations for given book size. BYTES may be suffixed
+with P, T, G, M or K to indicate a scaling factor.
+.RE
 .SH EXAMPLES
 .TP
 mem-reserve --show
 .TP
 mem-reserve --reserve 4T
+.TP
+mem-reserve --bsize 8G
 .SH AUTHOR
 .B Mem-reserve
 was written by Keith Packard <packard@hpe.com> and
```

Comment:
Add your name to the authoer list.

Reply:
Hmm, Keith came up with the idea and wrote the original, I'm not sure my small enhancement calls for an addition to the Authors list.

Reply:
It was the size of the code change, not the man page change ;  Your call.   Or is that what you're afraid of: hey this is broken, call Greg :-)